pkg/storage/unified/sql/backend.go

```diff
@@ -24,6 +31,15 @@ import (
 	"github.com/grafana/grafana/pkg/util/debouncer"
 )
 
+var ErrResourceAlreadyExists error = &apierrors.StatusError{
```

Comment:
I think we should lift this error outside the sql package because it should be generic across all backends (sql & spanner for now).